src/kafka_clj_utils/producers.clj

```diff
@@ -40,42 +40,49 @@
          value-ser       (avro-serializer/->avro-serializer serde-config)]
      (KafkaProducer. ^Map producer-config key-ser value-ser))))
 
-(defn- ->failure-tracking-callback [failure-state]
+(defn- ->callback [failure-state ack-callback-fn]
```

Comment:
Anyway to make `ack-callback-fn` optional? Will make upgrading the redeemer a bit easier

Reply:
Ignore that. Just spotted `publish-avro-bundle`